application/language/arabic/translations_lang.php

```diff
@@ -1,5 +1,8 @@
 <?php defined('BASEPATH') or exit('No direct script access allowed');
 // Arabic
+$lang['no_username_value_provided'] = 'No username value provided.';
```

Comment:
Please remove no_username_value_provided and no_password_value_provided and move invalid_credentials_provided to the end of the file before // End